portal-impl/src/portal.properties

```diff
@@ -6494,6 +6494,10 @@
     # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASED__MINUS__NUMBER2__NUMBER0__NUMBER2__NUMBER1__NUMBER3_
     #
     feature.flag.LPD-20213=false
+    
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASED__MINUS__NUMBER2__NUMBER1__NUMBER4__NUMBER1__NUMBER4_
+    #
+    feature.flag.LPD-21414=false
```

Comment:
Please include the FF name into the commit message, just in case we need to look for which commit was the one that introduced it